pkgs/by-name/lu/lubelog/package.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildDotnetModule
+, dotnetCorePackages
+, fetchFromGitHub
+}:
+
+buildDotnetModule rec {
+  pname = "lubelog";
+  version = "1.1.8";
+
+  src = fetchFromGitHub {
+    owner = "hargata";
+    repo = "lubelog";
+    rev = "v${version}";
+    hash = "sha256-yrNNNRTcbOf8OJ632VDhhjaLboUj+dpiHPwsihOyGf4=";
+  };
+
+  projectFile = "CarCareTracker.sln";
+  nugetDeps = ./deps.nix; # File generated with `nix-build -A package.passthru.fetch-deps`.
+
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.aspnetcore_8_0;
+
+  makeWrapperArgs = [
+    "--set DOTNET_CONTENTROOT ${placeholder "out"}/lib/lubelog"
+  ];
+
+  executables = [ "CarCareTracker" ]; # This wraps "$out/lib/$pname/foo" to `$out/bin/foo`.
+
+  packNupkg = true; # This packs the project as "foo-0.1.nupkg" at `$out/share`.
```

Comment:
Issues a warning:
```
/nix/store/1i9r9alwzyfflv6n5kpk9k1zacssm1hj-dotnet-sdk-8.0.101/sdk/8.0.101/Sdks/NuGet.Build.Tasks.Pack/build/NuGet.Build.Tasks.Pack.targets(203,6): warning : This project cannot be packaged because packaging has been disabled. Add <IsPackable>true</IsPackable> to the project file to enable producing a package from this project. [/build/source/CarCareTracker.csproj]
```
…and doesn't produce anything in `$out/share`.

```suggestion
```